viewer/README.md

```diff
@@ -131,6 +131,14 @@ Add the following script to your package's `package.json`:
 Running the command `yarn start` will host a localhost site with a template HTML that includes the `/app/index.ts` script that has been transpiled to javascript.
 To see an example of this check out the `packages/camera-manager` package.
 
+## Creating and running visual tests
+
+Visual test files must be on the format `visual-tests/SomeTest.VisualTest.ts`. See one of the existing tests
+for example on how to create a test.
+
+Visual tests can be run from `viewer/` using `yarn test:visual`. This will run all visual tests. You can also
+run a single test by using `yarn test:visual -- -t="SomeTest"` (will run all tests with "SomeTest" in the name).
```

Comment:
Technically all tests are run when starting the test server. What happens when multiple tests match to SomeTest now? Same as if no arg was passed?

Reply:
Also you should probably clarify what args are only passed to the test runner, not to the server.

Reply:
If multiple tests match, they are all ran.